docker-hub/access-tokens.md

```diff
@@ -7,62 +7,82 @@ keywords: docker hub, hub, security, PAT, personal access token
 
 Using personal access tokens provides some advantages over a password:
-* You can investigate when an access token was used last, and disable or delete it if you find any suspicious activity.
+* You can investigate the last usage of the access token and disable or delete it if you find any suspicious activity.
 * When logged in with an access token, you can't perform any admin activity on the account, including changing the password.
 
-Access tokens are also useful in building integrations, since you can issue
+>**Important**
+>
+> Treat access tokens like your password and keep them secret. Store your
+> tokens securely (for example, in a credential manager).
+{: .important}
+
+Access tokens are valuable for building integrations, since you can issue
 multiple tokens &ndash; one for each integration &ndash; and revoke them at
 any time.
 
-> Note: If you have [two-factor authentication (2FA)](/docker-hub/2fa) enabled on your account, you must create at least one personal access token. Otherwise, you will be unable to log in to your account from the Docker CLI.
-{: .important }
+   >**Note**
+   >
+   > If you have [two-factor authentication (2FA)](/docker-hub/2fa) enabled on
+   > your account, you must create at least one personal access token. Otherwise,
+   > you will be unable to log in to your account from the Docker CLI.
+
 
 ## Create an access token
 
 You can create as many tokens as you need.
 
-1. Log in to [hub.docker.com](https://hub.docker.com).
-
-2. Click on your username in the top right corner and select **Account
-Settings**.
+1. Log in to [hub.docker.com](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
 
-3. Select **Security > New Access Token**.
+2. Click on your username in the top right corner and select **[Account Settings](https://hub-stage.docker.com/settings/general)**.
 
-      ![](images/hub-create-token.png)
+3. Select **[Security](https://hub-stage.docker.com/settings/security) > New Access Token**.
 
 4. Add a description for your token. Use something that indicates where
-the token is going to be used, or set a purpose for the token.
+the token will be used, or set a purpose for the token.
+
+      ![PAT Menu](images/hub-create-token.png){:width="700px"}
+
+5. You can view the following permissions from the drop-down:
 
-5. Copy the token that appears on the screen. Make sure you do this now:
+    | Features   | Free | Pro | Team |
+    | --------------------- | ---- | ----- | ----- |
+    | Personal Access Tokens (read, write, delete scope only)*    |  x   |   x   |   x   |
+    | Scoped Personal Access Tokens**  |      | x   |   x   |
+    | Unlimited Personal Access Tokens   |      |       |   x   |
+
+      >If you would like more information about a Pro or a Team plan, see [Docker Pricing](https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade){: target="_blank" rel="noopener" class="_"}.
```

Comment:
could you change it to 'For more information...'

Reply:
Updated